generators/personabar/templates/Vue3JS/common/packages.config

```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="DotNetNuke.Abstractions" version="9.8.0" targetFramework="net48" />
-  <package id="DotNetNuke.Core" version="9.8.0" targetFramework="net48" />
-  <package id="DotNetNuke.DependencyInjection" version="9.8.0" targetFramework="net48" />
-  <package id="DotNetNuke.Instrumentation" version="9.8.0" targetFramework="net48" />
-  <package id="DotNetNuke.Web" version="9.8.0" targetFramework="net48" />
-  <package id="DotNetNuke.WebApi" version="9.8.0" targetFramework="net48" />
+  <package id="DotNetNuke.Abstractions" Version="9.11.0" targetFramework="net48" />
```

Comment:
@ArielBlanco1990 There should be a variable to use for this, instead of hard-coding the version number here. Please follow the pattern that was in place before this update. It previously used `dnnBuildVersionShort`. If that doesn't work and there's not a `dnnBuildVersionLong` or something, then we should have it created. Ideally, this update should be done in as few places as possible when we need to do it again in the future.

Reply:
@WillStrohl I followed your instructions and solved this problem. There was no `dnnBuildVersionLong `but there was `dnnBuildVersion `which is what we need. I replaced `<%= dnnBuildVersionShort %>` and `<%= dnnBuildVersion %>` in each case.